File: ConduitC#App/conduit-sample-app/config/AppConfig.cs
using System;
using System.Collections.Generic;

using NLog;

namespace config
{
    public class AppConfig
    {
        private static Logger logger = LogManager.GetCurrentClassLogger();
        private string email;
        private string password;
        private string authenticationType;
        private string dataSourceLocation;
        private string connectionUsername;
        private string typeName;
        private List<string> tables;
        private int partitionColumnsCount;
        private Boolean isAuthorizationEnabled;
        private string tableName;
        private string connectorName;
        private string token;
        static string url;

        public AppConfig()
        {
            email = getConfig("EMAIL");
            password = getConfig("PASSWORD");
            authenticationType = getConfig("AUTHENTICATION_TYPE");
            dataSourceLocation = getConfig("DATA_SOURCE_LOCATION");
            connectionUsername = getConfig("CONNECTION_USERNAME");
            typeName = getConfig("TYPE_NAME");
            tables = new List<string>(getConfig("TABLES").Split(','));
            partitionColumnsCount = Int32.Parse(getConfig("PARTITION_COLUMNS_COUNT"));
            isAuthorizationEnabled = Boolean.Parse(getConfig("IS_AUTHORIZATION_ENABLED"));
            tableName = getConfig("TABLE_NAME");
            connectorName = getConfig("CONNECTOR_NAME");
            token = getConfig("TOKEN");
            url = getConfig("URL");
        }

        public string getEmail()
        {
            return email;
        }

        public string getPassword()
        {
            return password;
        }

        public string getAuthenicationType()
        {
            return authenticationType;
        }

        public string getDataSourceConnecion()
        {
            return dataSourceLocation;
        }

        public string getConnectionUsername()
        {
            return connectionUsername;
        }

        public string getTypeName()
        {
            return typeName;
        }

        public List<string> getTables()
        {
            return tables;
        }

        public int getPartitionCount()
        {
            return partitionColumnsCount;
        }

        public Boolean getIsAuthorizationEnabled()
        {
            return isAuthorizationEnabled;
        }

        public string getTableName()
        {
            return tableName;
        }

        public string getConnectorName()
        {
            return connectorName;
        }

        public string getToken(){
            return token;
        }

        public static string getURL(){
            return url;
        }
        public static String getConfig(String config)
        {
            String configValue = Environment.GetEnvironmentVariable(config);
            if (configValue == null)
            {
                logger.Error(config + "is null. Please insert it.");
                Environment.Exit(1);
            }
            return configValue;
        }
    }

}

File: ConduitJavaApp/src/main/java/com/conduit/sample/config/AppConfig.java
package com.conduit.sample.config;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import java.util.Arrays;
import java.util.List;

public class AppConfig {
    private static final Logger LOGGER = LoggerFactory.getLogger(AppConfig.class);

    private String email;
    private String password;
    private String tokenUser;

    private String authenticationType;
    private String dataSourceLocation;
    private String connectionUsername;
    private String description;
    private String connectorName;

    private String typeName;
    private List<String> tables;
    private int partitionColumnsCount;
    private Boolean isAuthorizationEnabled;

    private String userJDBC;
    private String passJDBC;
    private String jdbcUrl;

    private String url;


    public AppConfig() {
        this.email = AppConfig.getConfig("EMAIL");
        this.password = AppConfig.getConfig("PASSWORD");
        this.tokenUser = AppConfig.getConfig("TOKEN");
        this.authenticationType = AppConfig.getConfig("AUTHENTICATION_TYPE");
        this.dataSourceLocation = AppConfig.getConfig("DATA_SOURCE_LOCATION");
        this.connectionUsername = AppConfig.getConfig("USERNAME");
        this.description = AppConfig.getConfig("DESCRIPTION");
        this.connectorName = AppConfig.getConfig("CONNECTOR_NAME");
        this.typeName = AppConfig.getConfig("TYPE_DATABASE");
        String stringTables = AppConfig.getConfig("TABLES");
        this.tables = Arrays.asList(stringTables.split(","));
        this.partitionColumnsCount = Integer.parseInt(AppConfig.getConfig("PARTITION_COLUMNS_COUNT"));
        this.isAuthorizationEnabled = Boolean.parseBoolean(AppConfig.getConfig("IS_AUTHORIZATION"));
        this.userJDBC = AppConfig.getConfig("USER_JDBC");
        this.passJDBC = AppConfig.getConfig("PASSWORD_JDBC");
        this.url = AppConfig.getConfig("URL");
        this.jdbcUrl = AppConfig.getConfig("JDBC_URL");
    }

    public String getEmail() {
        return email;
    }

    public String getPassword() {
        return password;
    }

    public String getAuthenticationType() {
        return authenticationType;
    }

    public String getDataSourceLocation() {
        return dataSourceLocation;
    }

    public String getConnectionUsername() {
        return connectionUsername;
    }

    public String getDescription() {
        return description;
    }

    public String getConnectorName() {
        return connectorName;
    }

    public String getTypeName() {
        return typeName;
    }

    public List<String> getTables() {
        return tables;
    }

    public int getPartitionColumnsCount() {
        return partitionColumnsCount;
    }

    public Boolean getAuthorizationEnabled() {
        return isAuthorizationEnabled;
    }

    public String getTokenUser() {
        return tokenUser;
    }

    public String getUserSQL() {
        return userJDBC;
    }

    public String getPassSQL() {
        return passJDBC;
    }

    public String getUrl() { return url;}

    public String getJdbcUrl() { return jdbcUrl; }

    public static String getConfig(String config) {
        String configValue = System.getProperty(config);
        if (configValue.isEmpty() && !config.equals("TOKEN")) {
            LOGGER.warn(config + " is null. Please insert it.");
        }
        return configValue;
    }
}
